src/components/ErrorBoundary.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+/**
+ * Helper component for debugging. Should not be used always
+ */
+export default class ErrorBoundary extends React.Component<any, any> {
```

Comment:
Does this have to be a class component?

Reply:
Yeah it has to be. It uses some of the lifestyle features which are only available in class components.